api/kyverno/v1/image_verification_types.go

```diff
@@ -456,7 +457,7 @@ func (ska *StaticKeyAttestor) Validate(path *field.Path) (errs field.ErrorList)
 }
 
 func (ca *CertificateAttestor) Validate(path *field.Path) (errs field.ErrorList) {
-	if ca.Certificate == "" && ca.CertificateChain == "" {
+	if ca.Cert == "" && ca.CertChain == "" {
```

Comment:
Was this change intentional? Does it relate to the field descriptions?

Reply:
the field `Certificate` and `CertificateChain` are called `Cert` and `CertChain` externally. To update the description, I updated the name of the field as well so the field name matches the variable name

https://github.com/kyverno/kyverno/pull/8779/files#diff-1fddc4e1393ac2e8bc54a080c1f43e856f9918aec60ae21697a6753cb5e25a6fR199

Reply:
I have reverted them to avoid any confusion